Remove unused 'using' statements in `VisibleUnits.cpp`

Ahmad-S792 opened the pull request:
#### 31cd3d537460e07881fc14e0b846a06e074b4f38
<pre>
Remove unused &apos;using&apos; statements in `VisibleUnits.cpp`

<a href="https://bugs.webkit.org/show_bug.cgi?id=272523">https://bugs.webkit.org/show_bug.cgi?id=272523</a>

Reviewed by Tim Nguyen.

This patch removes unused &apos;using&apos; statements from &apos;VisibleUnits.cpp&apos;.

* Source/WebCore/editing/VisibleUnits.cpp:
(previousLineCandidatePosition): Update call to use &apos;HTMLNames::brTag&apos;

Canonical link: <a href="https://commits.webkit.org/277388@main">https://commits.webkit.org/277388@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/686a9c313373fe459067018b9cad8808a9b8646f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/47456 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/26639 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/50118 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/50139 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/43504 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/49763 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/32255 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/24098 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/38635 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/48037 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/24232 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/40899 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/19957 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/21676 "Passed tests") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/42068 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/5499 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/43799 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/42484 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/52017 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/22490 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/18816 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/45939 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/23762 "Built successfully") | | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/44977 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/10474 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/24552 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/23480 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

whsieh commented on Source/WebCore/editing/VisibleUnits.cpp:
```diff
@@ -52,9 +52,6 @@
 
 namespace WebCore {
 
-using namespace HTMLNames;
```
Hm.. I think this file references `brTag`, which requires the namespace.

Ahmad-S792 replied:
@whsieh - Should I undo this line or update call-site to `HTMLNames::brTag`?

whsieh replied:
Either works for me! Up to you.

(It probably still builds right now due to source unification)

Ahmad-S792 replied:
I will update call-site. :-)

Ahmad-S792 replied:
@whsieh - done. :-)

Commit:
Remove unused 'using' statements in `VisibleUnits.cpp`

https://bugs.webkit.org/show_bug.cgi?id=272523

Reviewed by Tim Nguyen.

This patch removes unused 'using' statements from 'VisibleUnits.cpp'.

* Source/WebCore/editing/VisibleUnits.cpp:
(previousLineCandidatePosition): Update call to use 'HTMLNames::brTag'

Canonical link: https://commits.webkit.org/277388@main
